sdk/src/main/java/com/horizen/account/api/rpc/types/EthereumTransactionView.java

```diff
@@ -26,8 +26,11 @@ public class EthereumTransactionView {
     public final byte[] input;
     public final BigInteger gas;
     public final BigInteger gasPrice;
+    @JsonInclude(JsonInclude.Include.NON_NULL)
```

Comment:
let's extend one of our Py or Unit tests and check these 3 fields are included only for eip-1559 tx.